### PR TITLE
Finished ROS2 installation Tutorial - Issue #22

### PR DESCRIPTION
**Pull request by sampreets3**
* (Almost) finished ROS2 tutorial - Minor changes to be made in the "setting up environment" section and "Install Dependencies" section
* Thought it would be good to include an intermediate post between the two ROS installs, highlighting the need to shift from ROS to ROS2. Started working on that

### CODE REVIEW

**Comment by guilyx** [comment]
Do you want to merge now or do you wat to finish the first bullet point first ?

**Comment by sampreets3** [comment]
We can push it now, I have finished the installation tutorial. I will add stuff later if I come across more information.

**Comment by guilyx** [comment]
What's your take on using ROS2 with ROS already installed ? Just source everytime the version that you want instead of put it raw in the bashrc ?

**Comment by sampreets3** [comment]
Yeah I mean since the ROS workspace is already sourced, I figured I would just source the ROS2 workspace only when I am working on it. Just to avoid conflict.

**Comment by guilyx** [comment]
It works on my computer, I did that too (intuitively) but not sure if it's best practice, we could ask LeBastard :+1:

File: _posts/2020-06-02-ros-2.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+Congratulations! You have now successfully configured ROS2 on your target system. However, in order to start using the system, you would want to follow the next step, which does a little bit of book keeping.
+
+### Install dependencies
+
+
+
+### Setup the Environment
+
+With the completion of the previous step, ROS2 is now installed in our **/opt/ros/dashing** directory. Now, since we are using ROS2 and ROS side-by-side, we will source the ROS2 workspace only when we need it. We cannot put both the setup files in the .bashrc file, this would only source the last entry.
```
Comment (guilyx) [inline _posts/2020-06-02-ros-2.md]: Add the shell script line to source ROS2

**Comment by sampreets3** [comment]
Sounds good to me :+1:

**Review by guilyx — CHANGES_REQUESTED** [review]

File: _posts/2020-06-02-ros-2.md (hunk 1 above)
Comment (sampreets3) [reply _posts/2020-06-02-ros-2.md]: Did it in commit 882f8ec

**Comment by guilyx** [comment]
LGTM

**Review by guilyx — APPROVED** [review]

**Pull request merged**